lib/subscription.ex

```diff
@@ -152,4 +152,26 @@ defmodule Braintree.Subscription do
   def retry_charge(subscription_id, amount \\ nil) do
     Transaction.sale(%{amount: amount, subscription_id: subscription_id})
   end
+
+  @doc """
+  To update a subscription, use its ID along with new attributes. The same
+  validations apply as when creating a subscription. Any attribute not passed will
+  remain unchanged.
+  ## Example
```

Comment:
Please add some whitespace above and below the `##` here. I don't believe it is parsed correctly otherwise.